docker-compose.yaml

```diff
@@ -7,3 +7,23 @@ services:
     ports:
       - "8080:8080"
     command: uvicorn openapi_server.main:app --host 0.0.0.0 --port 8080
+    volumes:
+      - "./openapi_server:/venv/lib/python3.7/site-packages/openapi_server"
+    depends_on:
+      - localstack
+  localstack:
+    container_name: "${LOCALSTACK_DOCKER_NAME-localstack_main}"
+    image: localstack/localstack
+    ports:
+      - "127.0.0.1:4566:4566"            # LocalStack Gateway
+      - "127.0.0.1:4510-4559:4510-4559"  # external services port range
```

Comment:
Could you specify the only ports we use? dynamo and s3? This would be helpful to figure what exact ports we use and what they are.

Reply:
After some experimenting and searching the docs, it seems like dynamo and s3 exist on the port 4566 along with all the other localstack services. I removed the line `- "127.0.0.1:4510-4559:4510-4559"` and everything still functions the same. The "external services" referenced in the file seems to be for other services we may or may not use as described [here](https://docs.localstack.cloud/references/external-ports/). Should I push up the changes where it's removed?

Reply:
Yes it's still good practice leaving the only ports used.